RFS/src/main/java/com/rfs/common/http/ConnectionContext.java

```diff
@@ -102,6 +124,14 @@ public static class TargetArgs implements Params {
             "--target-password" }, description = "Optional.  The target password; if not provided, will assume no auth on target", required = false)
         public String password = null;
 
+        @Parameter(names = {
+            "--target-sigv4-region" }, description = "The source sigv4 region, if not provided will assume no sigv4 on source", required = false)
+        public String sigv4Region = null;
+
+        @Parameter(names = {
+            "--target-sigv4-service-name" }, description = "The source sigv4 service name, if not provided will assume es if sigv4 region provided", required = false)
+        public String sigv4ServiceName = "es";
```

Comment:
Lets remove this, the service name shouldn't be an exposed parameter - as its tailored to the endpoint that is called, ES for the managed service, maybe something different for the serverless endpoints , etc.

Reply:
This is different for serverless, "aoss".

Should we instead have a parameter for cluster type? Standalone, Amazon OpenSearch Service, Amazon OpenSearch Serverless Service?

Reply:
I'd rather not have to ask - if its sigv4, it has to be an Amazon managed system.  🤞 we can determine AOS from AOSS without the user specifying anything.

Reply:
The only way i see we can accurately determine AOS from AOSS (consider custom domain names/dns redirection/proxy layers) is by sending a failing request to the server and parsing the reject body. Let me know if you like this approach/have a better idea.

Reply:
Discussed offline. We'll keep a parameter `aws-service-signing-name` to specify this